### DISCUSSION TITLE
React/Remix or Vue2/StoreFront ?

### DISCUSSION THREAD

**Discussion by LeaKitty** [vendurehq/vendure#1669] [Q&A]
Hi,

Which of the two technologies (React/Remix or Vue2/StoreFront) would be easier and faster to implement ?

**Comment by michaelbromley** [comment]
Hi!
Right now I'd say that the Vue Storefront integration is more complete - it already includes the account management parts which are still under development in the Remix starter. The downside with VSF is that it is a little more complex to work with - there's an extra layer of abstraction there due to the fact that it is designed to work with multiple e-commerce backends. So longer term, you might find the Remix starter better to work with.

**Reply by huwgit** [reply]
On that subject the Remix collections - products page  only returns a maximum number of products? I've got 50 but only showing 25. Where would I set/change this maximum?

**Reply by huwgit** [reply]
I set a fix for this -- utils/filtered-search-loader.ts I set the "take" value to 70. Needs pagination I know.

    const searchResultPromise = search(
        {
            input: {
                groupByProduct: true,
                term,
                **take: 70,**
                facetValueIds,
                collectionSlug: params.slug,
            },
        },
        { request },
    );
